src/Client.php

```diff
@@ -94,11 +94,25 @@ public function search(
     ): Search {
         /** @var SearchEndpoint $endpoint */
         $endpoint = $this->container->get('search');
+
+        $sort = [];
+        if (is_array($sortField) && !is_null($sortOrder)) {
+            throw new \InvalidArgumentException('$sortOrder should be NULL when $sortField is an array');
+        }
+
+        if (is_array($sortField) && is_null($sortOrder)) {
```

Comment:
I realised that here, at this point if $sortField is array $sortOrder will be null as the previous check has asserted that. So we can just check if $sortField is array.